### PR TITLE
Rename internal errors to s3errors

### LINKED ISSUE #786: Rename api/errors to api/s3errors
Using standard package name is never a good idea and it's all about S3 errors, thus `s3errors`.

### PR DESCRIPTION
**Pull request by smallhive**
close #786

### CODE REVIEW

File: syncTree.sh [hunk 1]
```diff
@@ -6,9 +6,10 @@ REVISION="feaa9eace7098c343598bf08fb50746a1e8d2deb"
 
 echo "tree service revision ${REVISION}"
 
-FILES=$(curl -s https://github.com/nspcc-dev/neofs-node/tree/${REVISION}/pkg/services/tree | sed -n "s,.*\"/nspcc-dev/neofs-node/blob/${REVISION}/pkg/services/tree/\(.*\.pb\.go\)\".*,\1,p")
+FILES=$(curl -s https://github.com/nspcc-dev/neofs-node/tree/${REVISION}/pkg/services/tree | jq | sed -n "s,.*\"pkg/services/tree/\(.*\.pb\.go\)\".*,\1,p")
 
 for file in $FILES; do
+  echo ${file}
```
Comment (roman-khimov) [inline syncTree.sh:12]: What problem does it solve?

File: syncTree.sh [hunk 2]
```diff
@@ -6,9 +6,10 @@ REVISION="feaa9eace7098c343598bf08fb50746a1e8d2deb"
 
 echo "tree service revision ${REVISION}"
 
-FILES=$(curl -s https://github.com/nspcc-dev/neofs-node/tree/${REVISION}/pkg/services/tree | sed -n "s,.*\"/nspcc-dev/neofs-node/blob/${REVISION}/pkg/services/tree/\(.*\.pb\.go\)\".*,\1,p")
+FILES=$(curl -s https://github.com/nspcc-dev/neofs-node/tree/${REVISION}/pkg/services/tree | jq | sed -n "s,.*\"pkg/services/tree/\(.*\.pb\.go\)\".*,\1,p")
```
Comment (roman-khimov) [inline syncTree.sh:9]: What's the effect?

File: api/s3errors/errors.go [hunk 3]
```diff
@@ -270,7 +270,7 @@ const (
 
 	ErrPostPolicyConditionInvalidFormat
 
-	//CORS configuration errors.
+	// CORS configuration errors.
```
Comment (roman-khimov) [inline api/s3errors/errors.go:273]: Not related, btw.

File: syncTree.sh (hunk 2 above)
Comment (smallhive) [reply syncTree.sh:9]: Without these changes syncTree doesn't work - it doesn't find any files in _REVISION_.
I tend to think, GH changed the response format a little bit:
* shorter files pass, removed `/nspcc-dev/neofs-node/blob/${REVISION}` from file path
* and JSON is not formatted - just one string.

Unfortunately, I don't know how it was before, but without `jq` sed finds only one file. With formatted JSON - all files. How I understand it checks input by each row

File: syncTree.sh (hunk 1 above)
Comment (smallhive) [reply syncTree.sh]: None, debug info, removed

File: api/s3errors/errors.go (hunk 3 above)
Comment (smallhive) [reply api/s3errors/errors.go:273]: gofmt

File: syncTree.sh (hunk 2 above)
Comment (smallhive) [reply syncTree.sh:9]:
```shell
$ REVISION="feaa9eace7098c343598bf08fb50746a1e8d2deb"

Without jq
$ curl -s https://github.com/nspcc-dev/neofs-node/tree/${REVISION}/pkg/services/tree | sed -n "s,.*\"pkg/services/tree/\(.*\.pb\.go\)\".*,\1,p"
types_neofs.pb.go

With jq
$ curl -s https://github.com/nspcc-dev/neofs-node/tree/${REVISION}/pkg/services/tree | jq | sed -n "s,.*\"pkg/services/tree/\(.*\.pb\.go\)\".*,\1,p"
service.pb.go
service_grpc.pb.go
service_neofs.pb.go
types.pb.go
types_neofs.pb.go

File: api/s3errors/errors.go (hunk 3 above)
Comment (roman-khimov) [reply api/s3errors/errors.go:273]: I know, but it's a whitespace change -> separate commit.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**